apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PoolRewrite/SessionPool.TargetedSessionPool.cs

```diff
@@ -29,6 +29,9 @@ namespace Google.Cloud.Spanner.V1.PoolRewrite
     public partial class SessionPool
     {
         // Note: Internal for test purposes.
+        // TODO:
+        // - Revisit naming for KnownSessionCount/ActiveSessionCount
```

Comment:
nit: Yes, a couple of things now.

- ActiveSessionCount is purely informative now, which is OK. I think the name is good.
- I would change KnownSessionCount for PooledSessionCount or ManagedSessionCount (even if they are active they are in the Pool, or managed by the Pool, at least that's how I better understand it)
- I would really change Options.MaximumActiveSessions to Options.MaximumPooledSessions or Options.MaximumManagedSessions.
- And I would really change Options.MinimumPooledSessions for Options.MinimumIdleSessions.

I've realized with this PR that those two Options' names have been bugging me since the start.

Reply:
Oh OK, I don´t know entirely what was there before and what´s new, definetely agree with minimum changes even if those options' names are not super straightforward.

And yes, totally ok with leaving this till the end.